modules/internal/ChapelArray.chpl

```diff
@@ -2293,7 +2295,17 @@ module ChapelArray {
              "  Actual domain is: ", this.domain);
     }
 
-    pragma "no doc"
+    /* Return an array declared over a new domain, provided that the new
+       domain is of the same rank and size as the original.
```

Comment:
Do we have documentation on array views that I can cross-reference when using the term *array view* ?